tensorflow/g3doc/tutorials/mnist/pros/index.md

```diff
@@ -190,11 +190,11 @@ accomplished by repeatedly running `train_step`.
 
 ```python
 for i in range(1000):
-  batch = mnist.train.next_batch(50)
+  batch = mnist.train.next_batch(100)
```

Comment:
Oh, I see that it's inconsistent with the beginners tutorial.  This is fine then.